packages/qemacs.rb

```diff
@@ -20,12 +20,12 @@ class Qemacs < Package
   def self.build
     system "sed -i 's,css.h,libqhtml/css.h,' html2png.c"
     system "sed -i 's/$(prefix)/$(DESTDIR)$(prefix)/g' Makefile"
-    system './configure', \
-           "--prefix=#{CREW_PREFIX}", \
-           '--disable-x11', \
-           '--disable-xv', \
-           '--disable-xrender', \
-           '--disable-html', \
+    system './configure',
+           "--prefix=#{CREW_PREFIX}",
+           '--disable-x11',
+           '--disable-xv',
+           '--disable-xrender',
+           '--disable-html',
            '--disable-png'
```

Comment:
Why not remove all the quotes too?

Reply:
I agree! I'll tackle that in a different PR.